### PR TITLE
3573 Script api changes mentioned in the Debrief Script Demo

### PR DESCRIPTION
**Pull request by saulhidalgoaular**
Supports #3573

### CODE REVIEW

**Comment by saulhidalgoaular** [comment]
Fixes #3573

File: org.mwc.debrief.scripting/src/org/mwc/debrief/scripting/wrappers/Tote.java [hunk 1]
```diff
@@ -79,6 +82,9 @@ public WatchableList getPrimaryTrack()
    * Method that returns the secondary tracks of the current plot.
    * 
    * @return Array of secondary list of the current plot.
+   * 		<br />
+   *         // @type MWC.GenericData.WatchableList
```
Comment (IanMayo) [inline org.mwc.debrief.scripting/src/org/mwc/debrief/scripting/wrappers/Tote.java]: @helenayele - I'm pretty sure this needs array square-brackets at the end:
````
 // @type MWC.GenericData.WatchableList
````

**Comment by saulhidalgoaular** [comment]
Please, don't push more changes on this branch. Use https://github.com/debrief/debrief/tree/ease_develop_merge or branch off https://github.com/debrief/debrief/tree/ease_develop

File: org.mwc.debrief.scripting/src/org/mwc/debrief/scripting/wrappers/Tote.java (hunk 1 above)
Comment (helenayele) [reply org.mwc.debrief.scripting/src/org/mwc/debrief/scripting/wrappers/Tote.java]: @IanMayo  Corrected!

**Comment by IanMayo** [comment]
This is a hard-working branch that supported the development of scripting.

But, it now contains changes that are unrelated to scripting, and it has some unresolved product target related issues.

**Comment by IanMayo** [comment]
Resolved in separate branch.

**Pull request closed without merging**